cmd/weaviate/README.md

```diff
@@ -8,8 +9,13 @@ tldr; use `cmd/weaviate-server` instead. This is experimental
 
 ## Setup
 
+If you're prompted "Do you want the application “weaviate-server” to accept incoming network connections?" by your firewall select "Allow", this is because the server accepts incoming client connections (in this case from the querier).
```

Comment:
Nice. thanks for this @nathanwilk7. Can we make it as markdown NOTE type?
[some thing like this](https://github.com/orgs/community/discussions/16925)